cmd/litcli/accounts.go

```diff
@@ -293,29 +348,59 @@ func removeAccount(ctx *cli.Context) error {
 	defer cleanup()
 	client := litrpc.NewAccountsClient(clientConn)
 
-	var accountID string
+	id, label, _, err := parseIDOrLabel(ctx)
+	if err != nil {
+		return err
+	}
+
+	req := &litrpc.RemoveAccountRequest{
+		Id:    id,
+		Label: label,
+	}
+	_, err = client.RemoveAccount(ctxb, req)
+	return err
+}
+
+// parseIDOrLabel parses either the id or label from the command line.
+func parseIDOrLabel(ctx *cli.Context) (string, string, cli.Args, error) {
+	var (
+		accountID string
+		label     string
+	)
 	args := ctx.Args()
 
 	switch {
-	case ctx.IsSet("id"):
-		accountID = ctx.String("id")
+	case ctx.IsSet(idName) && ctx.IsSet(labelName):
+		return "", "", nil, fmt.Errorf("either account ID or label " +
+			"must be specified, not both")
+
+	case ctx.IsSet(idName):
+		accountID = ctx.String(idName)
+
+	case ctx.IsSet(labelName):
+		label = ctx.String(labelName)
+
 	case args.Present():
 		accountID = args.First()
 		args = args.Tail()
-	default:
-		return fmt.Errorf("id argument missing")
-	}
 
-	if len(accountID) == 0 {
-		return fmt.Errorf("id argument missing")
-	}
-	if _, err := hex.DecodeString(accountID); err != nil {
-		return err
-	}
+		// Since we have a positional argument, we cannot be sure it's
+		// an ID. So we check if it's an ID by trying to hex decode it
+		// and by checking the length. This will break if the user
+		// chooses labels that are also valid hex encoded IDs. But since
+		// the label is supposed to be human-readable, this should be
+		// unlikely.
```

Comment:
nit: It shouldn't be possible for the user to set such labels, no?

Reply:
Good point, I added a check here: https://github.com/lightninglabs/lightning-terminal/blob/568d7ad508933df0bcb4603b48594d3d0558f57b/accounts/store.go#L119

Reply:
Ah yeah, awesome! I realize that my comment above was a bit unclear, but I ment to say that the part of this comment I've marked isn't necessary, and can be removed as it shouldn't be possible for users to set such labels :). Though feel free to ignore this, as this isn't blocking at all and very nit picky!